src/panels/lovelace/create-element/create-card-element.ts

```diff
@@ -54,4 +54,8 @@ const CARD_TYPES = new Set([
 ]);
 
 export const createCardElement = (config: LovelaceCardConfig) =>
-  createLovelaceElement("card", config, CARD_TYPES);
+  createLovelaceElement(
+    "card",
+    config.type === "entity-button" ? { ...config, type: "button" } : config,
```

Comment:
This will get messy when there are more conversions, plus you change the config of a user here (not that bad in this case).

We could keep an empty `entity-button` that extends `button`?

Reply:
Yeah, that was my second option in mind. Was hoping you would just be fine with making it a breaking change 😜

Reply:
Sorry :-)

Reply:
Who's idea was it to say Lovelace was 1.0? 😭